### DISCUSSION TITLE
I'm a noob and not sure what this part means

### DISCUSSION THREAD

**Discussion by CreatorOblivion99** [TwitchIO/TwitchIO#316] [Questions] [answered]
Hi there, 
I'm not quite sure how a certain part works " from . import models, http " 
It doesn't work for my own programming as I'm using this to learn and program a bot (credit will of course go to your project!)
Where and how can I find out a bit more about this " . " module or how this works programming wise?

Thankyou
Joseph

Edit: I believe I have seen the answer to this by looking at the folder structure more. It appears to me that using  " from . import ... " is simply saying, " from the same folder import ... "
It would be nice if this could be confirmed, but I have another question and will make that a separate discussion

**Comment by chillymosh** [comment]
You don't need to import our models or http stack at all, all of this is handled for you via the Client/Bot and the respected objects.

**Comment by IAmTomahawkx** [comment, marked as answer]
`from . import X` is something used in packages
You can make packages in your code by simply making a subfolder and putting files in it.
When you see such code, it's basically the same as `from <thismodule> import X`

Further reading: https://blog.finxter.com/python-dot-relative-import/ (do ignore the random "cheat sheet" popup on the site)
